src/app/public/src/modules/swagger/swagger.component.ts

```diff
@@ -0,0 +1,33 @@
+import { AfterViewInit, Component, Input, OnInit } from '@angular/core';
+const swagger = require ('swagger-ui-dist');
+
+@Component({
+  selector: 'stache-swagger',
+  templateUrl: './swagger.component.html'
+})
+export class StacheSwaggerComponent implements OnInit, AfterViewInit {
+  @Input()
+  public swaggerUrl: string;
+
+  private swaggerConfig = {
+    // tslint:disable-next-line:max-line-length
+    url: 'https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/json/petstore-expanded.json',
```

Comment:
I may be reading this incorrectly, but there doesn't seem to be a reason to provide this value (`url`), since it will be overwritten in `ngOnInit` (even if it's undefined)?